src/main/converter/opds.ts

```diff
@@ -176,15 +197,29 @@ export class OpdsFeedViewConverter {
             });
         }
 
+        if (feed.Metadata) {
+            page = {
+                numberOfItems: feed.Metadata.NumberOfItems,
+                itemsPerPage: feed.Metadata.ItemsPerPage,
+            };
+        }
+
         return {
             title,
             type,
             publications,
             navigation,
-            searchUrl: await this.getSearchUrlFromOpds1Feed(feed),
+            urls,
+            page,
         };
     }
 
+    private getUrlFromFeed(feed: OPDSFeed, linkRel: string): string {
```

Comment:
The return type of the `getUrlFromFeed()` function should be `string | undefined` (which indicates that there is a possibility that a `link` with the given `rel` has not been found).